wrappers/python/tests/nntile_core/test_tensor_norm_fiber.py

```diff
@@ -0,0 +1,97 @@
+# @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+#                              (Skoltech), Russia. All rights reserved.
+#                2023-present Artificial Intelligence Research Institute
+#                              (AIRI), Russia. All rights reserved.
+#
+# NNTile is software framework for fast training of big neural networks on
+# distributed-memory heterogeneous systems based on StarPU runtime system.
+#
+# @file wrappers/python/tests/nntile_core/test_tensor_norm_fiber.py
+# Test for tensor::add_scalar<T> Python wrapper
+#
+# @version 1.1.0
+
+import numpy as np
+import pytest
+
+import nntile
+
+config = nntile.starpu.Config(1, 0, 0)
+nntile.starpu.init()
+
+# Define mapping between numpy and nntile types
+Tensor = {
+    np.float32: nntile.tensor.Tensor_fp32,
+    np.float64: nntile.tensor.Tensor_fp64
+}
+
+# Define mapping between tested function and numpy type
+norm_fiber = {
+    np.float32: nntile.nntile_core.tensor.norm_fiber_async_fp32,
+    np.float64: nntile.nntile_core.tensor.norm_fiber_async_fp64
+}
+
+norm_fiber_inplace = {
+    np.float32: nntile.nntile_core.tensor.norm_fiber_inplace_async_fp32,
+    np.float64: nntile.nntile_core.tensor.norm_fiber_inplace_async_fp64
+}
+
+
+@pytest.mark.parametrize(
```

Comment:
this parametrize can be split into 3 parametrize functions. Each parametrizes only a single parameter. Pytest then "multiplies" all combinations.